packages/engine/src/scene/components/debug/SplineHelperComponent.ts

```diff
@@ -75,39 +64,43 @@ export const SplineHelperComponent = defineComponent({
 
   onSet: (entity, component, json) => {
     if (!json) return
-    if (typeof json.layerMask == 'number') component.layerMask.set(json.layerMask)
+    if (typeof json.layerMask === 'number') component.layerMask.set(json.layerMask)
   },
 
   reactor: function () {
     const entity = useEntityContext()
     const component = useComponent(entity, SplineHelperComponent)
     const spline = useComponent(entity, SplineComponent)
-    const [sphereGeometry] = useResource(() => new SphereGeometry(0.05, 4, 2), entity)
-
-    const [greenMat] = useResource(greenMeshMaterial, entity)
-    const [greenSphere] = useDisposable(
-      Mesh,
-      entity,
-      sphereGeometry.value as SphereGeometry,
-      greenMat.value as MeshBasicMaterial
-    )
-
-    const [redMat] = useResource(redMeshMaterial, entity)
-    const [redSphere] = useDisposable(
-      Mesh,
-      entity,
-      sphereGeometry.value as SphereGeometry,
-      redMat.value as MeshBasicMaterial
-    )
+
+    const [lineGeometry] = useResource(createLineGeom, entity)
+    /** @todo these are probably unnecessary and were just used for debugging the implementation */
```

Comment:
Everything looks good without these, do we want to keep it in as a comment?

Reply:
yeah we can keep this as a todo to optimize or just come back to in the future